Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/CustomTabUtil.java

```diff
@@ -0,0 +1,17 @@
+package com.mentalmachines.droidcon_boston.utils;
+
+import android.content.Context;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsIntent;
+import com.mentalmachines.droidcon_boston.R;
+
+public class CustomTabUtil {
```

Comment:
unused?

Reply:
I didn't want to completely remove the code added by another dev, so I moved it into a Util method which is being used in the next PR for social screen via the new branch.